dojo/tools/hydra/parser.py

```diff
@@ -0,0 +1,75 @@
+import json
+
+from dojo.models import Finding, Endpoint
+from django.utils.dateparse import parse_datetime
+
+
+class HydraScanMetadata:
+    def __init__(self, generator):
+        self.date = generator['built']
+        self.command = generator['commandline']
+        self.schema_version = generator['jsonoutputversion']
+        self.service_type = generator['service']
+        self.tool_version = generator['version']
+        self.server = generator['server']
```

Comment:
are these fields always present? a lot of parser fail at some point due to missing fields.

Reply:
Yes, according to tool documentation those should always be there.

Reply:
I think we usually instruct authors to use something like `self.date = generator.get('built')` to reduce the chances of future exceptions on missing/renamed fields.

Reply:
Adjusted it accordingly - if some vital information is missing for findings (even though that should not happen) a warning is logged and that finding skipped